src/components/Channel/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Channel';
+export * from './hooks';
```

Comment:
call each of the hooks individually like https://github.com/GetStream/stream-chat-react/pull/713/files#diff-7fd43b0b162a214d655a8176a345b983bfa23ff95be8e4059e587cf9db3fc51f so we can get rid of a bunch of `index.ts/ts` files eventually